### PR TITLE
fix assigners

### PR DESCRIPTION
**Pull request by JSabadin**
This PR adjusts the assigners to function in the same manner.

### CODE REVIEW

**Review by kozlov721 — APPROVED** [review]

File: luxonis_train/assigners/tal_assigner.py [hunk 1]
```diff
@@ -254,4 +254,9 @@ def _get_final_assignments(
             torch.full_like(assigned_scores, 0),
         )
 
+        assigned_labels = torch.where(
```
Comment (klemen1999) [inline luxonis_train/assigners/tal_assigner.py:257]: Doesn't ATTS assigner also need this?

**Review by klemen1999 — APPROVED** [review]

File: luxonis_train/assigners/tal_assigner.py (hunk 1 above)
Comment (JSabadin) [reply luxonis_train/assigners/tal_assigner.py:257]: It already has it. I removed that line in the instance segmentation commit, and it’s better to have that line inside the assigner rather than in the loss function.
